GameData/Kerbalism/System/Signal.cfg

```diff
@@ -62,7 +62,7 @@
 		antennaType = DIRECT
 		packetInterval = 0.2
 		packetSize = 1.35
-		packetResourceCost = 0.5
+		packetResourceCost = 0.375
```

Comment:
Would be better to set the EC value to the equivalent power usage of the type of antenna that is used. And change the amount of EC given to a Kerbal when EVA to balance with the O2 use, this can be done in the default profile